pom.xml

```diff
@@ -140,5 +140,22 @@
       </snapshots>
     </repository>
   </repositories>
-
+  <pluginRepositories>
```

Comment:
This was missing in the pom which was causing build issues when running the repackage task